src/main/scala/ai/privado/entrypoint/ScanProcessor.scala

```diff
@@ -492,7 +514,8 @@ object ScanProcessor extends CommandProcessor {
       getDataflowCache,
       getAuditCache,
       getS3DatabaseDetailsCache,
-      appCache
+      appCache,
+      getPropertyFilterCache
```

Comment:
This function is not needed, we can pass the propertyFilterCache, like we passed `appCache` here

Reply:
done.